### DISCUSSION TITLE
overfitting on my first try

### DISCUSSION THREAD

**Discussion by nebyu08** [mrdbourke/pytorch-deep-learning#829] [Q&A]
i have noticed a very huge difference between my result when first training the efffiecientnet0,your testing accuracy seems to be 0.85 but mine is 1.1111 what does this mean
![Screenshot 2024-02-16 143735](https://github.com/mrdbourke/pytorch-deep-learning/assets/58502166/2df3bb68-7f46-4526-ba0f-7aba3e8df93c)

**Comment by LuluW8071** [comment]
Need the code snippet of model to see whats happening 
By any chance maybe you forgot to place right `output_labels` in `EfficientNet` model

**Reply by nebyu08** [reply]
i think you are right thanks
